minitests/iostandard/dump_iobs.tcl

```diff
@@ -0,0 +1,30 @@
+proc dump_iobs {file_name} {
+
+    set fp [open $file_name w]
+    puts $fp "tile,site_name,site_type,clock_region,bank,is_bonded,is_clock,is_global_clock,is_vref"
+
+    foreach tile [get_tiles *IOB33*] {
+        foreach site [get_sites -of_objects $tile] {
+            set site_type [get_property SITE_TYPE $site]
+            set clock_region [get_property CLOCK_REGION $site]
+            set is_bonded [get_property IS_BONDED $site]
+
+            set pin [get_package_pins -of_objects $site]
+            set bank [get_property BANK $pin]
+            set pkg_pin [get_property NAME $pin]
+            set is_clock [get_property IS_CLK_CAPABLE $pin]
+            set is_global_clock [get_property IS_GLOBAL_CLK $pin]
+            set is_vref [get_property IS_VREF $pin]
```

Comment:
Should also dump `PIN_FUNCTION`

Reply:
But why? It's not needed in the test. Actually what is needed are pkg_pin, is_vref and site name to pkg_pin mapping.

Reply:
You will want to have special handling for at least one pin function, PUDC, see: https://github.com/SymbiFlow/prjxray/blob/master/utils/fasm2frames.py#L125

Reply:
Added skipping of PUDC pin site(s)